CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.99.2 - 2024-03-14 - Add escaping semicolon for TXT records
```

Comment:
Standard practice w/octodns modules is that the version bumps and changelog header update happen in a dedicated release PR. we can leave this in here for now since it doesn't look like the actual `__version__` bump happened.